app/assets/stylesheets/styles.scss

```diff
@@ -0,0 +1,27 @@
+h1,
+h2,
+h3 {
+  font-weight: lighter;
+}
+
+h1 a {
+  color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.